zebrium/README.md

```diff
@@ -0,0 +1,52 @@
+# Zebrium Root Cause as a Service
+ 
+## Overview
+ 
+When you know there's a problem and you're not sure what caused it, [Zebrium][1] will automatically show you the root cause directly on your Datadog dashboards. It works by using machine learning on logs - but you don't need to manually train it or set up any rules. And it achieves accuracy in less than 24 hours. 
+
+Using Zebrium is simple. When you're troubleshooting a problem, instead of digging around, just scroll to the Zebrium panel on your dashboard and look at the details of the corresponding detection.
```

Comment:
```suggestion
Using Zebrium is simple. When you're troubleshooting a problem, instead of digging around, simply add the Zebrium app to your Datadog dashboard and look at the details of the corresponding detection.
```